### DISCUSSION TITLE
cockpit on Ubuntu 22.04.4 LTS

### DISCUSSION THREAD

**Discussion by Federico90-sys** [cockpit-project/cockpit#20608] [Q&A]
Hello everyone,

I have a problem with Cockpit installation on one of my Ubuntu Server VMs.

After installing cockpit, when I log into the web console, I get an error and cannot manage softwares updates.

I attach a screen of the error screen.

Do you know how I can solve this problem?

Thanks to everyone!

![Screenshot 2024-06-17 131606](https://github.com/cockpit-project/cockpit/assets/173055457/bd6908fd-5da5-4a81-a6ea-0e0738b39723)
![Screenshot 2024-06-17 131446](https://github.com/cockpit-project/cockpit/assets/173055457/d367dd83-8a96-469b-96dc-f78cb10e2218)

**Comment by martinpitt** [comment]
Please see https://cockpit-project.org/faq.html#error-message-about-being-offline

**Comment by Federico90-sys** [comment]
Great!
I understand that the problem is docuted to the fact that Cockpit uses NetworkManager for interface management (it does not use Netplan as is the default on Ubuntu).
If I set my VM to use NetworkManager by default, can that solve the problem?

Thank

**Reply by martinpitt** [reply]
Yes, that will. That's quite intrusive, so your decision. The workaround in the FAQ only creates a dummy device in NM.
